kedro_airflow_k8s/operators/init_experiment_operator.py

```diff
@@ -0,0 +1,65 @@
+import logging
+
+from airflow.operators.python import PythonOperator
+from airflow.utils.decorators import apply_defaults
+
+
+class InitExperimentOperator(PythonOperator):
```

Comment:
Rename to `StartMlflowExperimentOperator` or similar. Let's make it obvious it's Mlflow related

Reply:
If classes are loaded in the same scope we can try putting them inside function of DAG definitions.

Reply:
Regarding the broken tests, there was update of transitive dependencies between fields, I believe it was kedro. It triggered sqlalchemy updated which turned to be incompatible with airflow. I fixed that, but also added deptree for better visibility on such issues.